images-list-daily

```diff
@@ -2,5 +2,6 @@
 # See images-list for more information
 #
 # LIST FORMAT: <SOURCE> <DESTINATION> <TAG>
+neuvector/compliance-config rancher/mirrored-neuvector-compliance-config latest
```

Comment:
Does this repo already exist? Does the bot account have access to this repo?

Reply:
@brandond This repo rancher/mirrored-neuvector-compliance-config is already exist, and also available in image-list. Planning to update this file when ever there is new update in the compliance.